test/e2e/specs/onboarding/ftme__build.ts

```diff
@@ -0,0 +1,207 @@
+	RestAPIClient,
+	SecretsManager,
+	DomainSearchComponent,
+	SignupPickPlanPage,
+	NewSiteResponse,
+	TestAccount,
+} from '@automattic/calypso-e2e';
+import { Page, Browser } from 'playwright';
+import { apiDeleteSite } from '../shared';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'FTME: Build with "Something else" category' ), function () {
+	const blogName = DataHelper.getBlogName();
+
+	let siteCreatedFlag: boolean;
+	let newSiteDetails: NewSiteResponse;
+	let page: Page;
+
+	beforeAll( async function () {
+		page = await browser.newPage();
+
+		const testAccount = new TestAccount( 'simpleSiteFreePlanUser' );
+		await testAccount.authenticate( page );
+	} );
+
+	describe( 'Create site', function () {
+		it( 'Navigate to /new', async function () {
+			await page.goto( DataHelper.getCalypsoURL( 'start' ) );
+		} );
+
+		it( 'Select a .wordpress.com domain name', async function () {
+			const domainSearchComponent = new DomainSearchComponent( page );
+			await domainSearchComponent.search( blogName );
+			await domainSearchComponent.selectDomain( '.wordpress.com' );
+		} );
+
+		it( `Select WordPress.com Free plan`, async function () {
+			const signupPickPlanPage = new SignupPickPlanPage( page );
+			newSiteDetails = await signupPickPlanPage.selectPlan( 'Free' );
+
+			siteCreatedFlag = true;
+		} );
+	} );
+
+	describe( 'Onboarding', function () {
+		let startSiteFlow: StartSiteFlow;
+
+		beforeAll( async function () {
+			startSiteFlow = new StartSiteFlow( page );
+		} );
+
+		it( 'Enter Onboarding flow', async function () {
+			await expect( page.waitForURL( /setup\/goals\?/ ) ).resolves.not.toThrow();
+		} );
+
+		it( 'Select "Promote" goal', async function () {
+			await startSiteFlow.selectGoal( 'Promote' );
+			await startSiteFlow.clickButton( 'Continue' );
+		} );
+
+		it( 'Select "Something else" category', async function () {
+			await startSiteFlow.enterVertical( 'Something else' );
+			await startSiteFlow.clickButton( 'Continue' );
+		} );
+	} );
+
+	describe( 'Build', function () {
+		const themeName = 'Stewart';
+
+		let startSiteFlow: StartSiteFlow;
+
+		beforeAll( async function () {
+			startSiteFlow = new StartSiteFlow( page );
+		} );
+
+		it( 'Select theme', async function () {
+			await startSiteFlow.selectTheme( themeName );
+			await startSiteFlow.clickButton( `Start with ${ themeName }` );
+		} );
+
+		it( 'Land in Home dashboard', async function () {
+			await page.waitForURL(
+				DataHelper.getCalypsoURL( `/home/${ newSiteDetails.blog_details.site_slug }` )
+			);
+		} );
+	} );
+
+	afterAll( async function () {
+		if ( ! siteCreatedFlag ) {
+			return;
+		}
+
+		const restAPIClient = new RestAPIClient(
+			SecretsManager.secrets.testAccounts.simpleSiteFreePlanUser
+		);
+
+		await apiDeleteSite( restAPIClient, {
+			url: newSiteDetails.blog_details.url,
+			id: newSiteDetails.blog_details.blogid,
+			name: newSiteDetails.blog_details.blogname,
+		} );
+	} );
+} );
+
+describe(
```

Comment:
**suggestion**: while phrased only as a "style guide", each spec file should only have [only one top-level](https://wpcalypso.wordpress.com/devdocs/test/e2e/docs/style_guide.md#only-one-top-level-describe-block) `describe` block. This is because Jest parallelizes & spawns threads based on spec files, not `describe` blocks as it was with the older Mocha-based framework. 

Could we consider any one of the following alternatives?
1. move this top-level `describe` block to a new file.
2. move this top-level `describe` block under the initial top-level `describe` block (note that this will inevitably affect the test duration of Pre-Release Tests).
3. reconsider the need to cover both "Something Else" and "Travel Agencies & Services" categories in the Build flow.

**question**
Related to the third suggestion, is there a reason why there is a need to cover both selections? As I see that you've implemented some unit test coverage for the `vertical_id` attribute, my stance is that it is unnecessary to have what is an otherwise [duplicate coverage](https://developer.shopware.com/docs/resources/guidelines/testing/e2e-best-practises) of the Build flow only to test whether `vertical_id` is set. 

Let me know what you think.

Reply:
> is there a reason why there is a need to cover both selections?

The reason is that users might see different results in the following step but I agree there is a lot of duplication. According to the metrics below, the number of users who selected non-empty categories (2,586) is more than the number of users who selected empty categories (1,699). Thus, I'll just leave the "Travel Agencies & Services" category in the Build flow.

![image](https://user-images.githubusercontent.com/13596067/188561683-1588fe61-d660-45e2-bb0a-e46bc9cf654d.png)
See the `calypso_signup_site_vertical_submit` event